src/components/common/SearchInput/SearchInput.tsx

```diff
@@ -1,18 +1,42 @@
+import type { SearchEvent } from '@faststore/sdk'
 import {
   formatSearchState,
   initSearchState,
   sendAnalyticsEvent,
 } from '@faststore/sdk'
-import { SearchInput as UISearchInput } from '@faststore/ui'
-import { navigate } from 'gatsby'
-import React from 'react'
-import Icon from 'src/components/ui/Icon'
-import useSearchHistory from 'src/sdk/search/useSearchHistory'
-import type { SearchEvent } from '@faststore/sdk'
 import type {
   SearchInputProps as UISearchInputProps,
   SearchInputRef,
 } from '@faststore/ui'
+import { SearchInput as UISearchInput } from '@faststore/ui'
+import type {
+  SearchSuggestionsQueryQuery,
+  SearchSuggestionsQueryQueryVariables,
+} from '@generated/graphql'
+import { gql } from '@vtex/graphql-utils'
+import { navigate } from 'gatsby'
+import type { ChangeEventHandler } from 'react'
+import { default as React, useRef, useState } from 'react'
+import Suggestions from 'src/components/search/Suggestions'
+import Icon from 'src/components/ui/Icon'
+import { request } from 'src/sdk/graphql/request'
+import useSearchHistory from 'src/sdk/search/useSearchHistory'
+import useOnClickOutside from 'src/sdk/ui/useOnClickOutside'
+import './search-input.scss'
```

Comment:
Just a [reminder](https://github.com/vtex-sites/base.store/pull/437#discussion_r840784667) that this import is already happening in `components.scss`.

```suggestion
```